README.md

```diff
@@ -173,6 +175,68 @@ For, Lisk Mainnet:
 
 Refer to the `op-geth` configuration [documentation](https://docs.optimism.io/builders/node-operators/management/configuration#op-geth) for detailed information about available options.
 
+#### Run op-reth
+
+Navigate to your `reth` directory and start service by running the command:
+
+For, Lisk Sepolia Testnet:
+
+```sh
+./target/release/op-reth node \
+  -vvv \
+  --datadir="$DATADIR_PATH" \
+  --log.stdout.format log-fmt \
+  --ws \
+  --ws.origins="*" \
+  --ws.addr=0.0.0.0 \
+  --ws.port=8546 \
+  --ws.api=debug,eth,net,txpool \
+  --http \
+  --http.corsdomain="*" \
+  --http.addr=0.0.0.0 \
+  --http.port=8545 \
+  --http.api=debug,eth,net,txpool \
+  --authrpc.addr=0.0.0.0 \
+  --authrpc.port=8551 \
+  --authrpc.jwtsecret=PATH_TO_JWT_TEXT_FILE \
+  --metrics=0.0.0.0:606 \
+  --chain=PATH_TO_NETWORK_GENESIS_FILE \
+  --disable-discovery \
+  --rollup.sequencer-http=https://rpc.sepolia-api.lisk.com \
```

Comment:
We could leave it as a variable but recommend the office sequencer endpoint from Lisk.